test_fms/field_manager/test_field_manager2.sh

```diff
@@ -78,8 +78,7 @@ cat <<_EOF > input.nml
 /
 _EOF
 
-test_expect_success "field manager functional" '
-  mpirun -n 2 ./test_field_manager
-'
+test_expect_success "field manager functional r4" 'mpirun -n 1 ./test_field_manager_r4'
+test_expect_success "field manager functional r8" 'mpirun -n 1 ./test_field_manager_r8'
```

Comment:
more of suggestion but for the existing test inputs it might be good to try to read in a non-zero value.

also is there a reason to run with less PE's?

Reply:
The test doesn't require multiple PE's so I put down 1.  I'll change it to 2.

Reply:
fms_mod requires input.nml and I didn't want to put down a random variable to read in... thought it might be misleading, suggesting as if that variable specification is required.